codalab/worker/worker_run_state.py

```diff
@@ -161,6 +165,25 @@ def _transition_from_PREPARING(self, run_state):
             - Start the docker container
         4- If all is successful, move to RUNNING state
         """
+
+        def symlink(target, source):
+            try:
+                os.symlink(target, source)
+            except OSError as e:
+                # Override the existing symlink if one already exists
```

Comment:
Why are all these cases necessary now?

Reply:
I initially had this helper function because of the `FileExistsError` when creating symlinks for nested paths. I figured out that a better way to do this is to create all the directories up to the parent of the path and then create a symlink for the path itself.